src/components/ResultsItem/ResultsItem.tsx

```diff
@@ -34,12 +34,14 @@ export const ResultsItem = ({
   <article className="result-item">
     {meta && (meta.type || meta.date) && (
       <div className="result-item__meta">
-        {meta.type && !['taxonomy_term'].includes(itemType) && (
+        {/* remove this for now as types are not correctly displayed in search result
```

Comment:
Minor: Fair enough - t if you have time , would be useful to create a ticket and add the ticket number in this comment for reference

Reply:
will do it.